osmosis-1/osmosis.zone_assets.json

```diff
@@ -5274,6 +5274,97 @@
       "osmosis_unlisted": true,
       "listing_date_time_utc": "2024-10-20T12:00:00Z",
       "_comment": "Stratos native token (STOS)"
+    },
+    {
+      "chain_name": "int3face",
+      "base_denom": "uint3",
+      "path": "transfer/channel-82819/uint3",
+      "categories": [
+        "bridges"
+      ],
+      "osmosis_verified": false,
+      "listing_date_time_utc": "2024-10-11T03:00:00Z",
+      "_comment": "Int3face token"
+    },
+    {
+      "chain_name": "int3face",
+      "base_denom": "factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/dogecoin-doge",
+      "path": "transfer/channel-82819/factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/dogecoin-doge",
```

Comment:
the ibc connect between int3face and osmosis must be registered at the Chain Registry https://github.com/cosmos/chain-registry/tree/master/_IBC

Reply:
Ok, created the PR -> https://github.com/cosmos/chain-registry/pull/5448